examples/basic/README.md

```diff
@@ -1,11 +1,11 @@
-# Basic example
-
-<!--
-The basic example should call the module(s) stored in this repository with a basic configuration.
-Note, there is a pre-commit hook that will take the title of each example and include it in the repos main README.md.
-The text below should describe exactly what resources are provisioned / configured by the example.
--->
+# End to end basic example
```

Comment:
All examples are end to end, so maybe update the title back to "Basic example" ?